### PR TITLE
Support english dish names

### PR DESCRIPTION
**Pull request by stormofice**
As english translations for dishes are not guaranteed, there is a fallback to the german version of the dish's name in place.
The alt-text of the dish's image is also set to the correct version.

**Further consideration is needed on how to visualize the missing native (english) language value.**
Currently this is done intransparently to the user, which may cause confusion, however I could not think of an easy way to share this information with the user.

### CODE REVIEW

File: src/components/today-overview/TodayOverview.tsx [hunk 1]
```diff
@@ -34,8 +41,11 @@ const TodayOverview = () => {
 
   const content =
     data &&
-    data.occurrencesByDate.map((elem) => {
-      return <Dish name={elem.dish.nameDe} key={elem.id} />;
+    data.occurrencesByDate.map(({ dish: { nameDe, nameEn }, id }) => {
+      // Fallback to german value if no english value is present
+      return (
+        <Dish name={locale == 'en' ? nameEn ?? nameDe : nameDe} key={id} />
```
Comment (GamerBene19) [inline src/components/today-overview/TodayOverview.tsx]: Please use `===` here instead of `==`

File: .gitignore [hunk 2]
```diff
@@ -43,3 +43,6 @@ yarn-error.log*
 **/public/sw.js.map
 **/public/workbox-*.js.map
 **/public/worker-*.js.map
+
+# jetbrains IDEs
+.idea/
```
Comment (GamerBene19) [inline .gitignore:48]: Please add this in a separate commit so it's clearer what is happening here.
I would not expect a commit titled `
Support english dish names` to change `.gitignore`.

**Review by GamerBene19 — CHANGES_REQUESTED** [review]

**Comment by GamerBene19** [comment]
> Further consideration is needed on how to visualize the missing native (English) language value.

There could be a tooltip that is shown on hover that explains that there is no English name for this dish. However that does only work on desktops.

Other suggestion: A small text below the dish name indicating that there is no English name.

File: src/components/today-overview/TodayOverview.tsx (hunk 1 above)
Comment (stormofice) [reply src/components/today-overview/TodayOverview.tsx]: Done

File: .gitignore (hunk 2 above)
Comment (stormofice) [reply .gitignore:48]: Done

**Comment by stormofice** [comment]
I would favour your second suggestion, as it feels like tooltips are on their way out (no one seems to use them anymore).

Should this be implementend in this PR, because I think there may also be some styling involved in getting this to work?

**Review by GamerBene19 — APPROVED** [review]
Lgtm

**Comment by GamerBene19** [comment]
Sorry, overlooked this reply
> Should this be implementend in this PR, because I think there may also be some styling involved in getting this to work?

I think this is something that be added in the future. Can you create an issue for it?

**Comment by stormofice** [comment]
Sure!

**Pull request merged**